skunk/skunk.cu

```diff
@@ -156,6 +156,7 @@ extern "C" int scanhash_skunk(int thr_id, struct work* work, uint32_t max_nonce,
 					uint32_t secNonce = work->nonces[1] = startNounce + h_resNonce[1];
```

Comment:
already set here

Reply:
The code looks like setting the value for secNonce, while the change uses its value on a assignment. - Not the same thing